docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -36,17 +38,41 @@ C:\> sc query aesmservice
 The state of the service should be "running" (4). See the Troubleshooting
 section on how to manually install the PSW if it is not installed.
 
-In addition, you will need to set up a development environment to build both the
-Linux and Windows binaries:
+Microsoft Visual Studio 2017
+---------------------------------
+Install the latest version of [Microsoft Visual Studio 2017](https://www.visualstudio.com/downloads/).
+Visual Studio 2017's cmake support is required for building the Open Enclave SDK.
+For more information about cmake support, refer to
+https://blogs.msdn.microsoft.com/vcblog/2016/10/05/cmake-support-in-visual-studio/
 
-- [Windows Subsystem for Linux (WSL)](
-  https://docs.microsoft.com/en-us/windows/wsl/install-win10)
-- [Microsoft Visual Studio 2017](https://www.visualstudio.com/downloads/)
-  - [Windows 10 SDK (10.0.16299)](
-    https://developer.microsoft.com/en-US/windows/downloads/windows-10-sdk)
-- [Git for Windows 64-bit](https://git-scm.com/download/win)
-- [CMake 3.13.1+](https://cmake.org/download/)
-- [OCaml on Windows 64-bit](https://fdopen.github.io/opam-repository-mingw/installation/) (Note that this will install a Cygwin environment for OCaml.)
+
+Git for Windows 64-bit
+---------------------------------
+Install Git and add Git's bash to the path.
+Typically, Git's bash is located in C:\Program Files\Git\bin.
+Currently the Open Enclave SDK build system uses bash scripts to configure
+and process few 3rd-party libraries. 
+
+Open a command prompt and ensure that bash is available in the path:
```

Comment:
Is this a Git Bash window or a regular Command window? Please clarify.

Reply:
Does it matter?